### ISSUE TITLE
Question - debugging?

### ISSUE THREAD

**Issue by alexzaytsev-newsroomly** [tonerdo/pose#5]
I wrote a Pose test, but when i put breakpoints inside the action i pass to Pose or inside the code under test, none of my breakpoints are hit. (both using Resharper runner and VS built in runner)

I am using .Net core 2.0.

Is this expected?

**Comment by tonerdo** [comment]
@alexzaytsev-newsroomly Yes this is expected behavior. There's some work that needs to be done around debugging and exceptions

**Comment by ilengyel** [comment]
@tonerdo I suppose this also means that code coverage tools will not track lines called under the Isolate method?

**Comment by tonerdo** [comment]
@ilengyel yes that is correct

**Comment by douglasg14b** [comment]
What is the cause of this behavior? I'm curious to learn more.

**Comment by douglasg14b** [comment]
Maybe @Miista Has insight 🤔

**Comment by Miista** [comment]
Right. So the reason breakpoints are not hit is due to the code being rewritten at runtime. That is the short answer, at least. 

The slightly longer answer is that breakpoints are a feature of the IDE and not the IL. This means that information about a breakpoint is not available to us when rewriting the code. As such we cannot carry over the breakpoint. Even if we did have the information, I don't think there is functionality for setting a breakpoint in IL. 

@douglasg14b I hope that answers your question. If not please let me know.
